docs/faq.md

```diff
@@ -126,13 +126,13 @@ The default values are:
 
 ## How do I set up Alerting in PMM?
 
-When a monitored service metric reaches a defined threshold, PMM Server can trigger alerts for it either using the Grafana Alerting feature or by using an external Alertmanager, a high-performance solution developed by the Prometheus project to handle alerts sent by Prometheus.
+When a monitored service metric reaches a defined threshold, PMM Server can trigger alerts for it either using the Grafana Alerting feature or by using an external alert manager, such as [vmalert](https://github.com/VictoriaMetrics/VictoriaMetrics/tree/master/app/vmalert) by VictoriaMetrics.
```

Comment:
Alertmanager  must  stay, we just replace "to handle alerts sent by Prometheus"  to something like  "to handle alerts sent by vmalert"